docs/content/01_introduction/getting-started.md

```diff
@@ -163,3 +173,7 @@ ReactDOM.render(
   document.getElementById("root"),
 );
 ```
+
+## Running your application
```

Comment:
The capitalization should match the rest of the file. Either way is okay, it should just be consistent